sm/plat/generic/objects.mk

```diff
@@ -7,7 +7,12 @@ ifeq ($(PLATFORM),)
 	$(error PLATFORM not defined for SM)
 endif
 
-platform-genflags-y += "-DTARGET_PLATFORM_HEADER=\"platform/$(PLATFORM)/platform.h\""
+platform-genflags-y += "-DTARGET_PLATFORM_HEADER=\"platform/$(KEYSTONE_PLATFORM)/platform.h\""
```

Comment:
Since you're implementing a new sm platform, these changes should go into a subdirectory called `sm/plat/unmatched`. The `objects.mk` file in this directory will then be sourced by OpenSBI automatically as long as you define `BR2_TARGET_OPENSBI_PLAT="unmatched"` in your Buildroot config file

Reply:
Since OpenSBI requires generic platform to be specified as the default platform implementation for unmatched, the OpenSBI generic platform's `*.mk` must be included in `*.mk` under the `plat/unmatched`.
Is this acceptable?

Reply:
I see. Reading [here](https://github.com/carlosedp/riscv-bringup/blob/master/unmatched/Readme.md#build-opensbi), it does seem that the unmatched just uses the generic platform. Thanks for letting me know -- your implementation makes more sense to me now! I'll close my comments about this